packages/tools/examples/interpolationContourSegmentation/index.ts

```diff
@@ -304,10 +300,6 @@ function addBindings(toolGroupId) {
       {
         mouseButton: MouseBindings.Primary, // Left Click
       },
-      {
-        mouseButton: MouseBindings.Primary, // Shift + Left Click
-        modifierKey: KeyboardBindings.Shift,
-      },
     ],
```

Comment:
This will make the hole drawing not work, I guess Leo had some other PRs in hand to fix this so that we don't need to specify it here, right now when I want to draw a hole it does not work in examples @lscoder